upgrade/sql/8.0.0.sql

```diff
@@ -5,6 +5,29 @@ DROP TABLE IF EXISTS `PREFIX_referrer`;
 DROP TABLE IF EXISTS `PREFIX_referrer_cache`;
 DROP TABLE IF EXISTS `PREFIX_referrer_shop`;
 
+/* Remove page Referrers */
+## Remove Tabs
+DELETE FROM `PREFIX_tab` WHERE `class_name` = 'AdminReferrers';
+DELETE FROM `PREFIX_tab_lang` WHERE `id_tab` NOT IN (SELECT id_tab FROM `PREFIX_tab`);
+## Remove Roles
+DELETE FROM `PREFIX_access` WHERE `id_tab` NOT IN (SELECT id_tab FROM `PREFIX_tab`);
+/* For SalesMan profile, remove parent tab `Traffic & SEO` */
+DELETE FROM `PREFIX_access`
+  WHERE `id_authorization_role` IN (SELECT `id_authorization_role` FROM `PREFIX_authorization_role` WHERE `slug` LIKE 'ROLE_MOD_TAB_ADMINPARENTMETA_%')
+  AND `id_profile` = 4;
+DELETE FROM `PREFIX_authorization_role`
+  WHERE `slug` LIKE 'ROLE_MOD_TAB_ADMINREFERRERS_%';
+## Remove Configuration
+DELETE FROM `PREFIX_configuration`
+  WHERE `name` IN ('PS_REFERRERS_CACHE_LIKE', 'PS_REFERRERS_CACHE_DATE');
+## Remove Quick Access
+DELETE FROM `PREFIX_quick_access_lang`
+  WHERE id_quick_access IN (
+    SELECT id_quick_access FROM `PREFIX_quick_access` WHERE link LIKE '%controller=AdminReferrers%'
+  );
+DELETE FROM `PREFIX_quick_access`
+  WHERE link LIKE '%controller=AdminReferrers%';
+
```

Comment:
Similarly to what was done in the previous request maybe we could generalize this with:
```sql
DELETE FROM `PREFIX_quick_access`
  WHERE link LIKE '%controller=AdminReferrers%';
DELETE FROM `PREFIX_quick_access_lang` WHERE `id_quick_access` NOT IN (SELECT id_quick_access FROM `PREFIX_quick_access`);
```

Reply:
Finally method will be implemented in https://github.com/PrestaShop/autoupgrade/pull/504/ 🎉